src/components/NavBar.jsx

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+
+const NavWrapper = styled.nav`
```

Comment:
You don't always have to use the name Wrapper. You could just name this styled component Nav for example.

Reply:
Ok, great! Roger that. Thank you very much!!